src/main/resources/db/migration/V17__drop_table_processed_manifests.sql

```diff
@@ -0,0 +1 @@
+DROP TABLE processed_manifests;
```

Comment:
Since this is not in production, lets remove this table and all migrations, we can fix schema_migrations manually in staging. This way we don't leave around these migrations forever.

Reply:
Yeah I'd prefer that actually.